src/views/v2/components/read-more.php

```diff
@@ -9,15 +9,14 @@
  *
  * @link {INSERT_ARTCILE_LINK_HERE}
  *
- * @version 4.9.10
- *
+ * @version TBD
  */
+
 ?>
+<span class="tribe-events-c-small-cta-hellip"> &hellip; </span>
```

Comment:
since this doesn't really have anything to do with the small cta component, I think the class name should be something related to the read more component.